data/json/end_screen.json

```
@@ -0,0 +1,64 @@
+[
+  {
+    "type": "end_screen",
+    "id": "death_tombstone",
+    "priority": 0,
+    "picture_id": "ascii_tombstone",
+    "condition": { "not": "u_is_alive" },
+    "added_info": [
+      [ [ 6, 8 ], "In memory of: <u_name>" ],
+      [ [ 13, 9 ], "Survived: <time_survived>" ],
+      [ [ 15, 10 ], "Kills: <total_kills>" ]
+    ],
+    "last_words_label": "Last Words:"
+  },
+  {
+    "type": "end_screen",
+    "id": "death_cross",
+    "priority": 1,
```

just to make mods being able to put their own death screens in between of vanilla one
```suggestion
    "priority": 100,
```